File: Modulo2-ObjectiveC-Lorena/Exercicio17.m
//
//  main.m
//  Exercicio17-C
//
//  Created by Lorena Rodrigues Bruno on 06/09/2018.
//  Copyright © 2018 Lorena Rodrigues Bruno. All rights reserved.
//

#import <Foundation/Foundation.h>

float calculaAreaTrapezio (float altura, float baseMaior, float baseMenor) {
    return ((baseMaior + baseMenor)* altura) /2.0;
}

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        float altura = 1.5, baseMaior = 3.0, baseMenor = 2.0, resultado;
        resultado = calculaAreaTrapezio(altura, baseMaior, baseMenor);
        printf("O trapézio de altura %.2f, base maior de tamanho %.2f e base menor de tamanho %.2f tem área igual a %.2f\n", altura, baseMaior, baseMenor, resultado);
    }
    return 0;
}

File: Modulo2-Lorena/exercicio17.swift
//
//  main.swift
//  Exercicio17
//
//  Created by Lorena Rodrigues Bruno on 24/08/2018.
//  Copyright © 2018 Lorena Rodrigues Bruno. All rights reserved.
//

import Foundation

func calculaAreaTrapezio (altura : Float, baseMaior : Float, baseMenor : Float) -> Float {
    let resultado : Float = ((baseMaior + baseMenor)*altura)/2
    return resultado
}
var altura : Float = 1.5
var baseMaior : Float = 3.0
var baseMenor : Float = 2.0
var resultado : Float = Float(0)

resultado = calculaAreaTrapezio(altura: altura, baseMaior: baseMaior, baseMenor: baseMenor)
print("O trapézio de altura \(altura), base maior de tamanho \(baseMaior) e base menor de tamanho \(baseMenor) tem área igual a \(resultado)")
